nexus-description.txt

```diff
@@ -138,12 +138,10 @@ One of the main goals for ParallaxGen is to allow texture artists to worry about
 [*]Download ParallaxGen and drop all files in the zip to a folder of your choice (make sure this is a new empty folder) [b]DO NOT INSTALL AS A MOD IN YOUR MOD MANAGER[/b]
 [*]If using mod organizer, add ParallaxGen.exe as an executable to your instance
 [*]If using vortex, you need to enable "Run in shell" and "Run detached" for the console to pop up correctly
-[*]If you do not use Skyrim SE Steam edition, set your game type using [code]-g[/code]. For example, if you are a Skyrim VR user, add a CLI argument -g "skyrimvr". See the "CLI Arguments" section below for all available game types.
-
+[*]If you do not use Skyrim SE Steam edition, set your game type using [b]-g.[/b] For example, if you are a Skyrim VR user, add a CLI argument [b]-g skyrimvr[/b]. See the "CLI Arguments" section below for all available game types.
 [*]Run ParallaxGen.exe from either MO2 or the folder (if using Vortex) and generate meshes
 [*]A file ParallaxGen_Output/ParallaxGen_Output.zip should exist which you can import into your mod manager of choice
-[*]These meshes should win every conflict, even lighting mods. The exception is DynDoLOD. DynDoLOD should win conflicts over ParallaxGen.
-
+[*]These meshes should win every conflict, even lighting mods. Also the DynDoLod resources should be overwritten, otherwise there would be no parallax effect on the meshes that the resources provide. ParallaxGen does not change the geometry, so the LOD models still match. The exception is DynDoLOD output. DynDoLOD output should win conflicts over ParallaxGen.
```

Comment:
Is this actually true? I'm not even sure what dyndolod resources has so I'm just checking

Reply:
Yes, see https://dyndolod.info/Help/Load-Overwrite-Orders